README.md

```diff
@@ -28,7 +28,7 @@ Credits:
 ### Getting started and learning more
 
 The [latest version of HNC is
-v0.8.0](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.8.0).
+v0.8.0](https://github.com/kubernetes-sigs/hierarchical-namespaces/releases/tag/hnc-v0.8.0).
```

Comment:
The releases are still in the old repo for now so I think we need to leave these as-is.

Reply:
Hmm yeah, I thought I left them all. Will fix.